includes/rest/class-inbox.php

```diff
@@ -361,7 +361,7 @@ public static function handle_create( $object, $user_id ) {
 			'comment_post_ID' => $comment_post_id,
 			'comment_author' => \esc_attr( $meta['name'] ),
 			'comment_author_url' => \esc_url_raw( $object['actor'] ),
-			'comment_content' => \wp_filter_kses( $object['object']['content'] ),
+			'comment_content' => addslashes( \wp_kses( $object['object']['content'], 'pre_comment_content' ) ),
```

Comment:
isn't it easier to directly add the array of allowed HTML as second param instead of using the filter?

Reply:
Alternatively, could temporarily unhook the (second) core call to `wp_kses`